tests/ha/multi_pod_failure/test_multi_data_pod_failure.py

```diff
@@ -1454,76 +1434,48 @@ def test_deletes_during_kpods_down(self):
         wr_resp = ()
```

Comment:
make tuple()

Reply:
> make tuple()

Will take care of this in last PR, (), [], {} so it won't create conflicts as there are multiple PR already in review